contrib/examples/actions/workflows/mistral-workbook-multiple-subflows.yaml

```diff
@@ -39,38 +35,31 @@ workflows:
             call_action_chain:
                 action: examples.echochain
 
-    internal_workflow:
+    wf1:
         type: direct
         input:
-            - name
+            - v1
         output:
             report: <% $.report %>
-        task-defaults:
-          on-error:
-            - fail
         tasks:
-            task1:
+            t1:
                 action: core.local
                 input:
-                    cmd: "sleep 3; echo task1 ok"
+                    cmd: "echo 'task1 <% $.v1 %>'; sleep 1"
                 publish:
-                    report: <% $.task1.stdout %>
+                    report: <% $.t1.stdout %>
 
-    internal_workflow_multiple_tasks:
+    wf2:
         type: direct
         input:
-            - vm_id
-            - cpu_cores
-            - memory_mb
-        task-defaults:
-          on-error:
-            - fail
+            - v1
+            - v2
         tasks:
-            task1:
+            t1:
```

Comment:
nit: names like t1 don't look cool in UI, I think giving longer names here will also enforce the naming convention pattern.